hs/test/Reach/Test_Proto.hs

```diff
@@ -16,15 +18,26 @@ main :: IO ()
 main = hspec spec_proto
 
 spec_proto :: Spec
-spec_proto = around (Warp.testWithApplication (pure . appStubServer False $ Just "sleep 2 && echo yes")) $ do
+spec_proto = around (Warp.testWithApplication app) $ do
   u <- runIO $ parseBaseUrl "http://localhost"
   m <- runIO $ newManager defaultManagerSettings
   let env p = mkClientEnv m $ u { baseUrlPort = p }
+  let run p x = runClientM x (env p) >>= either (fail . show) (pure . id)
 
   describe "Requesting `reach compile`" $ do
-    it "should return a `Res` (which we'll flesh out later)" $ \p ->
-      runClientM (cmd'' "compile" $ Req [] Nothing mempty) (env p) >>= (`shouldBe` Right Res)
+    it "with a bogus flag like --nope will fail" $ \p -> do
```

Comment:
(BTW, I don't think `--flag` specs really belong in Test_Proto.hs long-term... these just test my temporary stubs)